scania/corporate-ui-site
# Icons page/Add search function

**Describe pull-request**  
- On search input, show filtered icons

**Solving issue**  

Fixes: #104 

**How to test**  
1. Use search function on icons page
2. Check if filter search works in all browsers

**Screenshots**  
![image](https://user-images.githubusercontent.com/1199101/95191749-7a849280-07d1-11eb-9b5d-95483a02f4cc.png)
